### ISSUE TITLE
Rectification needed

### ISSUE THREAD

**Issue by Tord-Zhang** [LeiaInc/holopix50k#22]
Hi, thanks for releasing this useful dataset.
It seems that the left image and right image are not perfectly rectified? Any way to rectify these two views? 
Thanks

**Comment by Tord-Zhang** [comment]
Sorry, I may have misunderstood the problem. It seems there are negative disparity, which can cause some prediction failure in most latest stereo matching algorithm. Any way to solve this problem?

**Comment by andrii-tsarov-leia** [comment]
Yes, the holopix dataset is randomly converged (might have positive and negative disparities). There is no real way of making full original images converge back to infinity as that would require restoring information that was cropped out during convergence process.

But you can crop inner parts of the images and that would make them all having positive disparity. I think if you crop around 85%-90% of image width from every pair that should work.

**Comment by Tord-Zhang** [comment]
@andrii-tsarov-leia Hi， thanks for your response. I tried cropping the center part of the image and then do stereo matching. The result is still not good. And I also tried to rectify the image pair using uncalibrated stereo rectification algorithm. But still no good. I am not sure what's the problem.

**Comment by Tord-Zhang** [comment]
In this paper https://arxiv.org/pdf/2203.11483.pdf, the author get very good stereo matching results in holopix50k. It was claimed that pre-rectfication was conducted before test the stereo matching on holopix50k.

**Comment by PuneetKohli** [comment]
@Tord-Zhang It might be worth reaching out to those authors for their technique as well, and then share it here. As for negative disparity, I believe you could just swap the left and right image to get positive disparity (I forgot if this actually works?).

**Comment by ptoupas** [comment]
@Tord-Zhang have you by any chance contacted the authors of [this paper](https://arxiv.org/pdf/2203.11483), about the pre-rectfication they used for the holopix50k dataset?
